client/settings/fraud-protection/advanced-settings/index.js

```diff
@@ -170,9 +186,6 @@ const FraudProtectionAdvancedSettingsPage = () => {
 						<LoadableBlock isLoading={ isLoading } numLines={ 20 }>
 							<AddressMismatchRuleCard />
 						</LoadableBlock>
-						<LoadableBlock isLoading={ isLoading } numLines={ 20 }>
```

Comment:
Removed order velocity related code according to [this](https://a8c.slack.com/archives/C01B8KNUYSW/p1679314937335329) discussion.